internal/controller/swadmin/iam_client_bucket_test.go

```
@@ -0,0 +1,227 @@
+		}
+	}
+}
+
+// startBucketAccessIAM serves srv on an ephemeral port and returns the filer
+// HTTP host:port the operator passes to NewIAMClient (seaweedfs derives the
+// gRPC port as +10000).
+func startBucketAccessIAM(t *testing.T, srv *bucketAccessIAM) string {
+	t.Helper()
+	gsrv := grpc.NewServer()
+	iam_pb.RegisterSeaweedIdentityAccessManagementServer(gsrv, srv)
+	lis, err := net.Listen("tcp", "127.0.0.1:0")
```

golangci-lint (2.12.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)